crates/nu-command/src/platform/input/input_.rs

```diff
@@ -132,31 +133,46 @@ impl Command for Input {
 
             if suppress_output || numchar_exists {
                 crossterm::terminal::enable_raw_mode()?;
+                // clear terminal events
+                while crossterm::event::poll(Duration::from_secs(0))? {
+                    // If there's an event, read it to remove it from the queue
+                    let _ = crossterm::event::read()?;
+                }
```

Comment:
I am not sure if this might become problematic if you try to run the whole thing in a "scripted" way (e.g. through `pexpect` or similar tools) if the buffer gets filled quicker than a human would typically type it in.

Reply:
Good point, I'll make some tests !